pkg/operator/controller/controller_test.go

```diff
@@ -0,0 +1,56 @@
+package controller
+
+import (
+	"github.com/openshift/cluster-dns-operator/pkg/manifests"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+func TestEnsureDNSNamespaceLabels(t *testing.T) {
```

Comment:
```suggestion
func Test_namespaceLabelsChanged(t *testing.T) {
```

Reply:
Changed to TestDNSNamespaceLabelsChanged